core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2673,22 +2673,6 @@ fn handle_event_loop<T: UserEvent>(
       event, window_id, ..
     } => {
       let window_id = webview_id_map.get(&window_id);
-      // NOTE(amrbashir): we handle this event here instead of `match` statement below because
-      // we want to focus the webview as soon as possible, especially on windows.
-      if event == WryWindowEvent::Focused(true) {
-        let w = windows
-          .borrow()
-          .get(&window_id)
-          .and_then(|w| w.inner.clone());
-        if let Some(WindowHandle::Webview(webview)) = w {
-          // only focus the webview if the window is visible
-          // somehow tao is sending a Focused(true) event even when the window is invisible,
-          // which causes a deadlock: https://github.com/tauri-apps/tauri/issues/3534
```

Comment:
we might have a regression with this issue, we need to check it

Reply:
since the focusing will be done through WRY, there shouldn't be a dead lock. If I remember correctly, the dead lock was in tauri itself, right?

Reply:
i don't remember exactly lol that issue is so confusing with the 49857 problems.